script/script.js

```diff
@@ -0,0 +1,10 @@
+const navWrapper = document.querySelector(".hamburger-menu");
+const hamMenu = document.querySelectorAll(".line");
+const sideNav = document.querySelector(".side-nav");
+const Toggler = (e) => {
+  e.preventDefault();
+  hamMenu.forEach((a) => a.classList.toggle("active"));
+  sideNav.classList.toggle("hide");
+};
+
+navWrapper.addEventListener("click", Toggler);
```

Comment:
- [ ] As stated the project requirement When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears. But When I clicked on any of mobile menu options a correct part of the page is displayed but the mobile menu doesn't disappears. Kindly Adjust it.